.github/workflows/on-pull-request.yml

```diff
@@ -38,6 +38,22 @@ jobs:
         run: brm validate
         working-directory: ${{ needs.get-module-to-validate.outputs.module_dir }}
 
+  shellcheck:
+    name: runner / shellcheck
+    runs-on: ubuntu-latest
+    needs: get-module-to-validate
+    if: needs.get-module-to-validate.outputs.module_dir
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: shellcheck
+        uses: reviewdog/action-shellcheck@v1
+        with:
+          github_token: ${{ secrets.github_token }}
```

Comment:
Isn't this optional? By default ${{ github.token }} is used.

Reply:
Looks like we need a pat to get the functionality I'm looking for https://github.com/reviewdog/reviewdog#reporter-github-pullrequest-review-comment--reportergithub-pr-review